checkClock/app/views/home/index.html.erb

```diff
@@ -1,61 +1,51 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  <nav>
-    <div class="banner">
-      <div class="logo">
-        <div class="clock-container">
-          <div class="clock-inside">
-            <div class="minutes-logo"></div>
-            <div class="hours-logo"></div>
-          </div>
-        </div>
-      </div>
-      <div class="page-title">Check clock</div>
-    </div>
-    <div class="navigation">
-      <a href="#">Admin Tools</a>
-    </div>
-  </nav>
-  <main>
-    <div class="col-left">
-      <div class="date">
-        <p id="date">o</p>
-      </div>
+<nav class="home-nav">
+  <div class="banner">
+    <div class="logo">
```

Comment:
Can you put your banner in an external file?

Reply:
Yes, we will implement this recommendation in future commits